statsmodels/tsa/stattools.py

```diff
@@ -1735,6 +1735,282 @@ def _sigma_est_kpss(resids, nobs, lags):
+             (42.500, -3.37465), (45.000, -3.33394), (47.500, -3.29393),
+             (50.000, -3.25316), (52.500, -3.21244), (55.000, -3.17124),
+             (57.500, -3.13211), (60.000, -3.09204), (65.000, -3.01135),
+             (70.000, -2.92897), (75.000, -2.83614), (80.000, -2.73893),
+             (85.000, -2.62840), (90.000, -2.49611), (92.500, -2.41337),
+             (95.000, -2.30820), (96.000, -2.25797), (97.000, -2.19648),
+             (98.000, -2.11320), (99.000, -1.99138), (99.900, -1.67466), )
+        self._za_critical_values['t'] = np.asarray(self._t)
+        # constant + trend model
+        self._ct = ((0.001, -38.17800), (0.100, -6.43107), (0.200, -6.07279),
+              (0.300, -5.95496), (0.400, -5.86254), (0.500, -5.77081),
+              (0.600, -5.72541), (0.700, -5.68406), (0.800, -5.65163),
+              (0.900, -5.60419), (1.000, -5.57556), (2.500, -5.29704),
+              (5.000, -5.07332), (7.500, -4.93003), (10.000, -4.82668),
+              (12.500, -4.73711), (15.000, -4.66020), (17.500, -4.58970),
+              (20.000, -4.52855), (22.500, -4.47100), (25.000, -4.42011),
+              (27.500, -4.37387), (30.000, -4.32705), (32.500, -4.28126),
+              (35.000, -4.23793), (37.500, -4.19822), (40.000, -4.15800),
+              (42.500, -4.11946), (45.000, -4.08064), (47.500, -4.04286),
+              (50.000, -4.00489), (52.500, -3.96837), (55.000, -3.93200),
+              (57.500, -3.89496), (60.000, -3.85577), (65.000, -3.77795),
+              (70.000, -3.69794), (75.000, -3.61852), (80.000, -3.52485),
+              (85.000, -3.41665), (90.000, -3.28527), (92.500, -3.19724),
+              (95.000, -3.08769), (96.000, -3.03088), (97.000, -2.96091),
+              (98.000, -2.85581), (99.000, -2.71015), (99.900, -2.28767), )
+        self._za_critical_values['ct'] = np.asarray(self._ct)
+
+    def _za_crit(self, stat, model='c'):
+        """
+        Linear interpolation for Zivot-Andrews p-values and critical values
+
+        Parameters
+        ----------
+        stat : float
+            The ZA test statistic
+        model : {'c','t','ct'}
+            The model used when computing the ZA statistic. 'c' is default.
+
+        Returns
+        -------
+        pvalue : float
+            The interpolated p-value
+        cvdict : dict
+            Critical values for the test statistic at the 1%, 5%, and 10%
+            levels
+
+        Notes
+        -----
+        The p-values are linear interpolated from the quantiles of the
+        simulated ZA test statistic distribution
+        """
+        table = self._za_critical_values[model]
+        y = table[:, 0]
+        x = table[:, 1]
+        # ZA cv table contains quantiles multiplied by 100
+        pvalue = np.interp(stat, x, y) / 100.0
+        cv = [1.0, 5.0, 10.0]
+        crit_value = np.interp(cv, y, x)
+        cvdict = {"1%" : crit_value[0], "5%" : crit_value[1],
+                  "10%" : crit_value[2]}
```

Comment:
no space before colon

are there other names besides x, y that might be more informative?  one-letter names make it tough to search a file for all-places-this-is-used

Reply:
Fixed the colon spacing and renamed to (pcnts, stats) rather than (x, y).